client/src/components/App/Content/InvestigationForm/TabManagement/TabManagement.tsx

```diff
@@ -3,90 +3,108 @@ import { Tabs, Tab, Card, createStyles, withStyles } from '@material-ui/core';
 import PersonalInfoTab from './PersonalInfoTab/PersonalInfoTab';
 import ClinicalDetails from './ClinicalDetails/ClinicalDetails';
 import InteractionsTab from './InteractionsTab/InteractionsTab';
 import ExposuresAndFlights from './ExposuresAndFlights/ExposuresAndFlights';
 
-export const defaultTab: TabObj = {
-    id: 0,
-    name: TabNames.PERSONAL_INFO,
-    isDisabled: false,
-    displayComponent: <PersonalInfoTab />
-};
 
 export const tabs: TabObj[] = [
-    defaultTab,
+    {
+        id: 0,
+        name: TabNames.PERSONAL_INFO,
+    },
     {
         id: 1,
         name: TabNames.CLINICAL_DETAILS,
-        isDisabled: false,
-        displayComponent: <ClinicalDetails />
     },
     {
         id: 2,
         name: TabNames.EXPOSURES_AND_FLIGHTS,
-        isDisabled: false,
-        displayComponent: <ExposuresAndFlights/>,
     },
     {
         id: 3,
         name: TabNames.INTERACTIONS, 
-        isDisabled: false,
-        displayComponent: <InteractionsTab/>,
     },
 ];
 
 const TabManagement: React.FC<Props> = (tabManagementProps: Props): JSX.Element => {
-    const { currentTab, setCurrentTab, onTabClicked, shouldDisableChangeTab } = tabManagementProps;
+
+    const {
+        currentTab,
+        moveToNextTab,
+        setCurrentTab,
+        setNextTab
+    } = tabManagementProps;
+
     const classes = useStyles({});
     const StyledTab = withStyles((theme) =>
         createStyles({
             root: {
                 fontWeight: theme.typography.fontWeightRegular,
             },
+            wrapper: {
+                flexDirection: "row-reverse",
+            }
         }),
     )(Tab);
 
-  const handleTabChange = (event: React.ChangeEvent<{}>, selectedTab: number) => {
-    setCurrentTab({
-        id: selectedTab,
-        name: tabs[selectedTab].name, 
-        displayComponent: tabs[selectedTab].displayComponent,
-        isDisabled: false,
-    });
-  };
+    const investigationId = useSelector<StoreStateType, number>((state) => state.investigation.epidemiologyNumber);
+    const formsValidations = useSelector<StoreStateType, any>((state) => state.formsValidations[investigationId]);
 
     return (
         <Card className={classes.card}>
                 <Tabs
-                    value={currentTab.id}
+                    value={currentTab}
                     indicatorColor='primary'
                     textColor='primary'
-                    onChange={(event, selectedTab) => !shouldDisableChangeTab && handleTabChange(event, selectedTab)}
                 >
                     {
                         tabs.map((tab) => {
-                            return <StyledTab onClick={onTabClicked} key={tab.id} label={tab.name} disabled={tab.isDisabled}/>
+                            if (formsValidations !== undefined && !formsValidations[tab.id] && formsValidations[tab.id] !== null) {
+                                return <StyledTab 
+                                // @ts-ignore
```

Comment:
why @ts-ingore?

Reply:
We need to give 'form' attribute to each tab, Material ui doesn't like it when we prop form attribute so @ts-ignore fixed it

Reply:
so it links the tab to the form?

Reply:
yes